project/funcoes.class.js

```diff
@@ -0,0 +1,29 @@
+// Classe para chamar o Json.
+function json(){
+	var qtd;
+	var retorno;
+
+	// Resgatar valores.
+	json.prototype.resgatarValores = function(){
+		$('#resultado').html('Carregando dados...');
+
+		// Estrutura de resultado.
+		$.getJSON('urls.json', function(data){
+			this.qtd = data.links.length;
+			this.retorno = '';
+			for (i = 0; i < this.qtd; i++){
+                this.retorno += '<table><tr>';
```

Comment:
Tks for the feedback, i w'll pratice more and learning the doc's about jquery and json. In this challenge, the hardest for me was the jquery and json. I dont have a lot of pratice, but i try.